prov/tcp/src/tcpx_ep.c

```diff
@@ -550,8 +550,8 @@ static int tcpx_ep_close(struct fid *fid)
 	tcpx_ep_tx_rx_queues_release(ep);
 	tcpx_progress_ep_del(ep);
```

Comment:
@vkrishna - what is the intent of tcpx_progress_ep_del()?

Reply:
tcpx_progress_ep_add/del adds/deletes the active endpoint socket to the cq wait object.  This is useful for apps which use cq_sread.

Reply:
Separate from this change, can you rename the function to better reflect that?

Reply:
sure will do.